libs/db-browser/src/main/java/com/oceanbase/tools/dbbrowser/schema/mysql/MySQLNoGreaterThan5740SchemaAccessor.java

```diff
@@ -164,10 +164,10 @@ public void switchDatabase(String schemaName) {
     @Override
     public List<DBObjectIdentity> listUsers() {
         MySQLSqlBuilder sb = new MySQLSqlBuilder();
-        sb.append("SELECT DISTINCT user FROM mysql.user");
+        sb.append("SELECT DISTINCT user,host FROM mysql.user");
         return jdbcOperations.query(sb.toString(), (rs, rowNum) -> {
             DBObjectIdentity dbUser = new DBObjectIdentity();
-            dbUser.setName(rs.getString(1));
+            dbUser.setName(rs.getString(1) + "@'"+ rs.getString(2)+"'");
```

Comment:
we may add a field to hold host rather than append host message to the user's name

Reply:
we can regard 'user'%'host' as a whole username, like : `CREATE USER 'username'@'hostname' IDENTIFIED BY 'password'`;

Reply:
it is different, what do do is to change the behavior and that is forbidden unless it is necessary~

Reply:
done, instand of query user from OscDBAccessor